packages/experiments/src/components/Stack/Stack.styles.ts

```diff
@@ -27,7 +26,7 @@ export const styles: IStackComponent['styles'] = (props, theme): IStackStylesRet
     padding,
     horizontalAlign,
     verticalAlign,
-    shrinkItems,
+    preventShrink,
```

Comment:
Still sounds a little unpredictable in what it might do, but I don't have a better suggestion.